wilkes/roster.json

```diff
@@ -33,6 +33,10 @@
         {
           "type": "video",
           "url": "sgczYwGzumk"
+        },
+        {
+          "type": "portfolio,
```

Comment:
You're missing the closing double quote :)

Reply:
My Bad, Thank you for the catch, I think the auto checks may have actually pointed out as well but I don't think I realised.